pyaedt/generic/constants.py

```diff
@@ -181,6 +181,24 @@ def scale_units(scale_to_unit):
     return sunit
 
 
+def validate_enum_class_value(cls, value):
+    """Return bool whether value is a valid value for enumeration-class cls.
+
+    Parameters
+    ----------
+    cls : class
+        Enumeration-style class with integer members in range(0, N) where cls.Invalid equals N-1.
+    value : int
+        value to check
+
+    Returns
+    -------
+    bool
+        Whether value is a valid value for enumeration class cls.
+    """
+    return isinstance(value, int) and value >= 0 and value < cls.Invalid
```

Comment:
are you sure that an enum in ironputhon is classified as an int? I've noticed that pythonnet3.0 return an enum class and the isinstance(value, int) would return false